tools/cockpit.spec

```diff
@@ -709,6 +709,11 @@ Requires: libvirt-client
 Requires: libvirt-dbus >= 1.2.0
 # Optional components
 Recommends: virt-install
+Recommends: libosinfo-devel
+%if 0%{?rhel} == 7
+Recommends: python-gobject-base
```

Comment:
RHEL/CentOS 7 don't yet support "Recommends". Also, this addition is in the `%if 0%{?fedora} || 0%{?rhel} >= 8` section, so just simply add the Recommends: --  Let's not force these on RHEL 7, it's ok if auto-detection does not work there yet.

-devel is not necessary, the typelib is in libosinfo already (as it should be).

Reply:
Mind that listing the OSes in the vm-create dialog would also be broken in case the deps are not installed. With that in mind, wouldn't be better to have a Requires: entry for el7?